### PR TITLE
JSON Exporter

### PR DESCRIPTION
**Pull request by bentonam**
Added support for JSON Exporter to auto-probe via the following annotations: 

```
json.agent.grafana.com/probe
json.agent.grafana.com/port
json.agent.grafana.com/path
json.agent.grafana.com/module
json.agent.grafana.com/tenant
json.agent.grafana.com/job
json.agent.grafana.com/interval
json.agent.grafana.com/timeout
```

Also updated the following modules to exporter the remote writer: 

- `modules/kubernetes/metrics/all.river`
- `modules/kubernetes/logs/all.river`
- `modules/kubernetes/logs/events.river`
- `modules/kubernetes/logs/kubelet.river`

Updated the following target modules to be more generic for probes instead of specific to blackbox: 

- `modules/kubernetes/metrics/targets/services.river`
- `modules/kubernetes/metrics/targets/ingresses.river`

### CODE REVIEW

File: example/kubernetes/logs/simple-events.river
Comment (erikbaranowski) [inline example/kubernetes/logs/simple-events.river:10]: good catch

File: modules/kubernetes/README.md
Comment (erikbaranowski) [inline modules/kubernetes/README.md:64]:
```suggestion
| `json.agent.grafana.com/probe` | Boolean whether or not to probe the service / ingress for metrics. *Note*: If a pod exposes multiple ports, all ports would be probed. To limit this behavior specify the port annotation to limit the probe to a single port. |
```

**Review by erikbaranowski — APPROVED** [review]
LGTM! this is admittedly out of my practical knowledge for a deeper review so if @mattdurham wants to take a look for another set of eyes before merge I would welcome it.

File: example/kubernetes/metrics/json-exporter.river
Comment (mattdurham) [inline example/kubernetes/metrics/json-exporter.river:27]: Can we add links to the JSON Exporter?

Comment (bentonam) [reply example/kubernetes/metrics/json-exporter.river:27]: They do exist in the actual modules 

```
./modules/kubernetes/metrics/scrapes/json-services.river
./modules/kubernetes/metrics/scrapes/json-ingresses.river
```

I'll add them to the example

**Review by mattdurham — APPROVED** [review]
LGTM

**Pull request merged**
